catalogue/webapp/pages/works.js

```diff
@@ -433,8 +388,8 @@ const Works = ({ works, searchParams }: Props) => {
 };
 
 Works.getInitialProps = async (ctx: Context): Promise<Props> => {
+  const workTypeInUrl = ctx.query.workType;
```

Comment:
All this should be dealt by with the parsers - fancy going through what's going on here? I can't quite get my head around all these edge cases?